dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -22,12 +20,6 @@ public sealed class SKContext
     /// <returns>Processed input, aka result</returns>
     public string Result => this.Variables.ToString();
 
-    /// <summary>
-    /// When a prompt is processed, aka the current data after any model results processing occurred.
-    /// (One prompt can have multiple results).
-    /// </summary>
-    public IReadOnlyCollection<ModelResult> ModelResults { get; set; } = Array.Empty<ModelResult>();
```

Comment:
Can you please throw an [Obsolete] on this one, with instructions guiding users where to find the new version?

(We don't need to add obsoletes everywhere, but this is one I know some people will hit)

You can just throw NotImplementedException instead of get/set, as long as there are instructions telling callers the new way of accessing this.

Reply:
Thanks, done!